### PR TITLE
Updates to Nodes page

### PR DESCRIPTION
**Pull request by sg00dwin**
- restructure columns and add Role
- Add role column
- Move node addresses into the expanded view, along with labels

design doc: http://openshift.github.io/openshift-origin-design/web-console/future-openshift/nodes/nodes

### CODE REVIEW

File: frontend/public/components/factory/list.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ import { ingressValidHosts } from '../ingress';
 import { routeStatus } from '../routes';
 import { secretTypeFilterReducer } from '../secret';
 import { bindingType, roleType } from '../RBAC';
+// import { getRole } from '../node';
```
Comment (sg00dwin) [inline frontend/public/components/factory/list.tsx]: @spadgett we tried to import `getRole` to sort by, but it didn't work. Should that have worked?

**Comment by spadgett** [comment]
/hold

@sg00dwin this should go to master-next. you should rebase against current master-next

**Pull request merged**